src/protocols/Pager/Pager.cpp

```diff
@@ -28,6 +28,14 @@ PagerClient::PagerClient(PhysicalLayer* phy) {
   #if !RADIOLIB_EXCLUDE_DIRECT_RECEIVE
   readBitInstance = phyLayer;
   #endif
+
+  filterMask = 0xFFFF;
```

Comment:
If the intention is to initialize the mask to all ones, then it should be `0xFFFFF`, to be 20 bits.

Reply:
Yep.